packages/commerce-sdk-react/src/hooks/ShopperBaskets/cache.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+    ShopperBaskets,
+    ShopperBasketsTypes,
+    ShopperCustomers,
+    ShopperCustomersTypes
+} from 'commerce-sdk-isomorphic'
+import {ApiClients, Argument, CacheUpdate, CacheUpdateMatrix, MergedOptions} from '../types'
+import {and, matchesPath, matchParameters, pick} from '../utils'
+
+type Client = ApiClients['shopperBaskets']
+/** Data returned by every Shopper Baskets endpoint (except `deleteBasket`) */
+type Basket = ShopperBasketsTypes.Basket
+/** Data returned by `getCustomerBaskets` */
+type CustomerBasketsResult = ShopperCustomersTypes.BasketsResult
+/** Parameters that get passed around, includes client config and possible parameters from other endpoints */
+type BasketParameters = MergedOptions<Client, Argument<Client['getBasket']>>['parameters']
+/** Parameters that we actually send to the API for `getBasket` */
+type GetBasketParameters = Argument<ShopperBaskets<{shortCode: string}>['getBasket']>['parameters']
```

Comment:
why is shortCode so special here?

Reply:
It's a required parameter for all API clients. Can't do anything without it.